src/client/dfuse/dfuse_core.c

```diff
@@ -893,19 +884,15 @@ dfuse_cont_open(struct dfuse_projection_info *fs_handle, struct dfuse_pool *dfp,
 		if (fs_handle->dpi_info->di_caching) {
 			rc = dfuse_cont_get_cache(dfc);
 			if (rc == ENODATA) {
-				/* If there is no container specific
-				 * attributes then use defaults
-				 */
-				DFUSE_TRA_INFO(dfc,
-					"Using default caching values");
+				/* If there is no container specific attributes then use defaults */
```

Comment:
grammar nitpick - only if you need to push again
```suggestion
				/* If there are no container specific attributes then use defaults */
```